### PR TITLE
MPR#7048: ocamldoc -latex, don't escape Latin-1 accented vowels

### PR DESCRIPTION
**Pull request by xavierleroy**
This causes trouble when the doc comments are actually in UTF8 and
the user is providing their own preamble with an UTF8 inputenc.

The default preamble still contains `\usepackage[latin1]{inputenc}`,
so the Latin-1 accents will still display fine.

Original report: https://caml.inria.fr/mantis/view.php?id=7048

### CODE REVIEW

**Review by gasche — APPROVED** [review]
This is clearly correct.

**Review by Octachron — APPROVED** [review]
It also looks good to me.

File: Changes [hunk 1]
```diff
@@ -349,6 +349,9 @@ Release branch for 4.06:
   when no -title has been provided
   (Pierre Boutillier)
 
+- MPR#7048: ocamldoc, in -latex mode, don't escape Latin-1 accented vowels
```
Comment (Octachron) [inline Changes]: Utterly minor nitpicking: ç is not a vowel.

**Comment by xavierleroy** [comment]
> ç is not a vowel.

Touché!  I fixed the Changes file and will now merge.  Thanks for the quick feedback.

**Comment by xavierleroy** [comment]
Cherry-pick [4.06 6a8c0ec]

**Pull request merged**